### ISSUE TITLE
Avoid unnecessary linting

### ISSUE THREAD

**Issue by psychelzh** [MikhailArkhipov/vscode-r#61]
For example, this `UseMethod()` call is legal to create `S3` generic.

![image](https://user-images.githubusercontent.com/13306020/112492462-3e171400-8dbc-11eb-87d9-eba218817ebb.png)

And this should also be legal:

![image](https://user-images.githubusercontent.com/13306020/112492647-66067780-8dbc-11eb-8706-5e3811c3491f.png)

**Comment by MikhailArkhipov** [comment]
Linting is not about legal/not legal. All naming is legal. It is about _style_. Rules replicate basically what [lintr ](https://www.rdocumentation.org/packages/lintr/versions/1.0.3)does. You can turn on or off any specific checks per https://github.com/MikhailArkhipov/vscode-r/blob/master/README.md#linting. Try setting `r.lintingr.pascalCase` to `false`.

**Comment by MikhailArkhipov** [comment]
Perhaps there should be a feature to turn it off per line in a comment or something. Or have a file with suppressed names.

**Comment by psychelzh** [comment]
Yes, found this. `# nolint` should work line-wise, but it doesn't have effect in this tool.


> Lintr supports per-project configuration of the following fields. The config file (default file name: `.lintr`) is in Debian Control Field Format.
>
> * `linters` - see `?with_defaults` for example of specifying only a few non-default linters.
> * `exclusions` - a list of filenames to exclude from linting. You can use a named item to exclude only certain lines from a file.
> * `exclude` - a regex pattern for lines to exclude from linting. Default is "# nolint"
> * `exclude_start` - a regex pattern to start exclusion range. Default is "# nolint start"
> * `exclude_end` - a regex pattern to end exclusion range. Default is "# nolint end"
